### PR TITLE
feat: adds readrows closed start no end

### PR DESCRIPTION
**Pull request by telpirion**

### CODE REVIEW

File: tests/readrows_test.go [hunk 1]
```diff
@@ -447,3 +447,55 @@ func TestReadRows_NoRetry_MultipleRowRanges(t *testing.T) {
...
 }
+
+// TestReadRows_NoRetry_ClosedStartUnspecifiedEnd tests that the client can request
+// a row range with a closed start key and no end key.
+func TestReadRows_NoRetry_ClosedStartUnspecifiedEnd(t *testing.T) {
+	type testRow struct {
+		key, cf string
+	}
+
+	trs := []testRow{
```
Comment (liujiongxin) [inline tests/readrows_test.go:458]: I wonder if you can just make the keys and values inline (i.e., remove the struct definition and the struct array).

After all, there are just two rows. So the for loop on line 471 is less straightforward than what you did in the other test cases.

Comment (telpirion) [reply tests/readrows_test.go]: Done.

File: tests/readrows_test.go [hunk 2]
```diff
@@ -451,28 +451,17 @@ func TestReadRows_NoRetry_MultipleRowRanges(t *testing.T) {
...
 
 	rec := make(chan *readRowsReqRecord, 3)
 	var seq []*readRowsAction
-	for _, tr := range trs {
-		seq = append(seq, &readRowsAction{
-			chunks: []chunkData{dummyChunkData(tr.key, tr.cf, Commit)},
-		})
-	}
+	seq = append(seq, &readRowsAction{
+		chunks: []chunkData{dummyChunkData(keys[0], cfs[0], Commit)},
```
Comment (liujiongxin) [inline tests/readrows_test.go:460]: They can be included in `seq := []*readRowsAction{xx, yy}` directly so `append` is not needd.

**Review by liujiongxin — APPROVED** [review]

File: tests/readrows_test.go (hunk 2 above)
Comment (telpirion) [reply tests/readrows_test.go]: Done.

**Pull request merged**